.gitignore

```diff
@@ -1,4 +1,6 @@
 __pycache__
 .vscode
 env
-flask
\ No newline at end of file
+flask
+install.sh
+urllib.request
```

Comment:
Why you are ignoring these? are you using them anywhere?

Reply:
it's my fault. Actually **urllib.request** needed for the weather application.

Reply:
But you are not using it anywhere I guess

Reply:
I already check it, The **urllib.request** is used for this application.why because " The urllib.request module defines functions and classes which help in opening URLs (mostly HTTP) in a complex world — basic and digest authentication, redirections, cookies and more."

Reply:
Yes. Why are you ignoring it?

Reply:
Before, I don't know why it is using. That's why I ignored it.